core/src/Streamly/Internal/Data/Array/Unboxed/Type.hs

```diff
@@ -55,7 +55,9 @@ module Streamly.Internal.Data.Array.Unboxed.Type
     , toStream
     , toStreamRev
     , toList
-
+#ifdef DEVBUILD
+    , toStreamD_
+#endif
```

Comment:
We should not use conditional here unless that is really needed, this is an internal module we can export whatever we need from this.